routes/api/users.js

```diff
@@ -71,17 +70,33 @@ router.post("/users/login", function(req, res, next) {
 });
 
 router.post("/users", function(req, res, next) {
-    const user = new User();
+    // const user = new User();
 
-    user.username = req.body.user.username;
-    user.email = req.body.user.email;
-    user.setPassword(req.body.user.password);
+    // user.username = req.body.user.username;
+    // user.email = req.body.user.email;
+    // user.setPassword(req.body.user.password);
+
+    // user.save()
+    //     .then(function() {
+    //         return res.json({ user: user.toAuthJSON() });
+    //     })
+    //     .catch(next);
+
+    const username = req.body.username,
+    email = req.body.email,
+    hash= req.body.password;
+
+    User.create({ username, email, hash})
+    .then(function(user){
+        return res.status(201).json({ 
+            email: user.email,
+            token: 'This feature is yet to be developed',
+            username: user.username,
+            bio: user.bio || `Hi ${user.username}, please update your profile`,
+            image: user.image || `Don't forget to upload your profile image`,
```

Comment:
I believe your task/chore is to set up the application to use Postgres, can you explain what you are trying to achieve here?.

Reply:
Thanks for this feedback. I added that block to enable me to test that the database setup is working since we don't have our automated test framework setup.